simpl/schema/external/types.py

```diff
@@ -66,12 +74,21 @@ def resolve_status(obj, info):
 
     @staticmethod
     def resolve_players(obj, info):
-        users = User._default_manager.filter(character__instance=obj)
-        return utils.get_auth0_ids(*users)
+        try:
+            characters = utils.get_instance_characters(obj)
+            users = []
+            for character in characters:
+                users.extend([account.uid for account in character.user.auth0_accounts])
+        except (AttributeError, KeyError):
```

Comment:
Reading this code it is not immediately obvious when we would take the exception path here. Can you summarise when we hit this path? Maybe a good one to explain in a comment.

Reply:
Thanks for flagging this up. I've added a comment explaining what's going on. The summary is that we first try to get the players/users from the pre-fetched attribute `auth0_accounts`. If that's not found, we hit the exception and go the normal route of querying the db.